app/templates/catch-all.hbs

```diff
@@ -1,4 +1,4 @@
-<p id='flash' class='shown'>Oops, that route doesn't exist!</p>
+<p id='flash' class='show'>Oops, that route doesn't exist!</p>
```

Comment:
why did you change the class name?

Reply:
For me, the noun is made more sense here, because this is what this css class doing. But English is not my native language and maybe I'm wrong.